guides/source/testing.md

```diff
@@ -545,6 +545,28 @@ to be able to easily change the number of workers a test run should use:
 PARALLEL_WORKERS=15 rails test
 ```
 
+### Parallel Testing of Transactions
```

Comment:
I think this needs a different title since we have a feature called "parallel testing". How about "Testing Parallel Transactions
"?

Reply:
Sound good, thanks, I changed it.